chia/types/condition_opcodes.py

```diff
@@ -9,33 +9,39 @@ class ConditionOpcode(bytes, enum.Enum):
 
     # AGG_SIG is ascii "1"
 
-    # signature opcodes
```

Comment:
We need to make sure these don't affect the standard puzzle, have you confirmed this? 
We need to check this and actually spend coins that were farmed using old puzzle hashes.

Reply:
@richardkiss @wbendick @matt-o-how can each of you confirm that this does not affect existing puzzle hashes for standard puzzle?

Reply:
I recompiled all the puzzles, and got the same results, which suggest the only puzzles that change pre-images with this PR are the ones included in the PR.

Reply:
But we should definitely try to spend lots of coins in the test-mainnet or whatever it's called, especially the pre-farm.